### PR TITLE
Prepare for the United Kingdom leaving the European Union

### PR DESCRIPTION
**Pull request by jimmackenzie**
Following the guidance [here](https://www.privacyshield.gov/article?id=Privacy-Shield-and-the-UK-FAQs), a few tweaks to extend our Privacy Shield commitment to the United Kingdom, in preparation for the UK leaving the European Union.

I also added a link to our sub-processors, for consistency with our main privacy policy page.

### CODE REVIEW

File: privacy/privacy-shield.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Basecamp EU-US and Swiss-US Privacy Shield policy
 
-*Last updated: May 24, 2018*
+*Last updated: January 30, 2019*
 
 The privacy of your data—and it is your data, not ours!—is a big deal to us. We’ll only ever access your account to help you with a problem or squash a software bug. We’ll never open any uploaded files unless you ask us to. We log all access to all accounts by IP address, so we can always verify that no unauthorized access has happened for as long as the logs are kept.
 
-This policy applies to personal data from the European Union and from Switzerland that is collected, used, and retained by us in the United States.
+This policy applies to personal data from the European Union, the United Kingdom and from Switzerland that is collected, used, and retained by us in the United States.
```
Comment (jeremy) [inline privacy/privacy-shield.md]: "from A, B and from C" → "from A, B, and C" or "from A, from B, and from C" ?

File: privacy/privacy-shield.md [hunk 2]
```diff
@@ -58,18 +58,18 @@ In order to improve our services and the website, and provide more convenient, r
 ## Third Parties
 You understand that Basecamp uses third party vendors and hosting partners to provide the necessary hardware, software, networking, storage, and related technology required to provide you with our services. A current list of vendors is available upon request.
 
-In cases of onward transfer to these third parties for data of EU or Swiss individuals received pursuant to the Privacy Shield Frameworks, Basecamp is potentially liable should any issues or concerns arise.
+In cases of onward transfer to these third parties for data of EU, United Kingdom or Swiss individuals received pursuant to the Privacy Shield Frameworks, Basecamp is potentially liable should any issues or concerns arise.
```
Comment (jeremy) [inline privacy/privacy-shield.md]: Preference on clarity/ambiguity without Oxford comma?

File: privacy/privacy-shield.md [hunk 3]
```diff
@@ -58,18 +58,18 @@ In order to improve our services and the website, and provide more convenient, r
...
+Basecamp complies with the EU-U.S. Privacy Shield Framework and the Swiss-U.S. Privacy Shield Framework as set forth by the U.S. Department of Commerce regarding the collection, use, and retention of personal information transferred from the European Union and the United Kingdom and Switzerland to the United States, respectively. Basecamp has certified to the Department of Commerce that it adheres to the Privacy Shield Principles. If there is any conflict between the terms in this privacy policy and the Privacy Shield Principles, the Privacy Shield Principles shall govern. To learn more about the Privacy Shield program, and to view our certification, please visit <a href="https://www.privacyshield.gov/">https://www.privacyshield.gov/</a>.
```
Comment (jeremy) [inline privacy/privacy-shield.md]: "A and B and C" reads a bit awkwardly. Go with commas?

File: privacy/privacy-shield.md (hunk 1 above)
Comment (jimmackenzie) [reply privacy/privacy-shield.md]:
```suggestion
This policy applies to personal data from the European Union, the United Kingdom, and Switzerland, which is collected, used, and retained by us in the United States.
```

File: privacy/privacy-shield.md (hunk 2 above)
Comment (jimmackenzie) [reply privacy/privacy-shield.md]:
```suggestion
In cases of onward transfer to these third parties for data of EU, United Kingdom, or Swiss individuals received pursuant to the Privacy Shield Frameworks, Basecamp is potentially liable should any issues or concerns arise.
```
Yep, that's better 👍🏼

File: privacy/privacy-shield.md (hunk 3 above)
Comment (jimmackenzie) [reply privacy/privacy-shield.md]: Mmm, yes. The model language uses the repeated and, but this seems like a place where serial commas would work nicely.

```suggestion
Basecamp complies with the EU-U.S. Privacy Shield Framework and the Swiss-U.S. Privacy Shield Framework as set forth by the U.S. Department of Commerce regarding the collection, use, and retention of personal information transferred from the European Union, the United Kingdom, and Switzerland to the United States, respectively. Basecamp has certified to the Department of Commerce that it adheres to the Privacy Shield Principles. If there is any conflict between the terms in this privacy policy and the Privacy Shield Principles, the Privacy Shield Principles shall govern. To learn more about the Privacy Shield program, and to view our certification, please visit <a href="https://www.privacyshield.gov/">https://www.privacyshield.gov/</a>.
```

**Pull request merged**